README.md

```diff
@@ -71,6 +89,106 @@ To get around this, you can create an
 points your project to the copy of react that it should be using or you can
 simply remove `react` and `react-dom` from `react-plotly.js-editor`.
 
+### Using the `simple` example for development
+
+In one terminal:
+
+```
+npm link       # registers react-plotly.js-editor for subsequent npm link calls
+npm run watch  # keep this running
+```
+
+In another terminal:
+
+```
+cd examples/simple # or any other application directory
+npm install
+rm -rf node_modules/react node_modules/react-dom node_modules/react-plotly.js-edit
+npm link react-plotly.js-edit
+npm start # keep this running
+```
+
+## Built-in Components
+
+This module provides a number of nestable _containers_ which are intended to contain _fields_ that render _widgets_ that have been connected to individual values in the figure via _connector functions_. Containers can also be connected to certain parts of the figure tree such that their child fields map to the appropriate leaf values. A field must have a connected container as an ancestor in order to be bound to the figure. The `<PlotlyEditor />` and connector functions use the React [`context` API]() to push configuration information to child components.
```

Comment:
This is a pretty heavy paragraph. I see you discuss bound and unbound components below so maybe we should keep this here but perhaps we can try and draw a line between "typical" usage of the library vs "advanced" (rolling your own components). 

We don't want users to freak out about HOCs and context when all they are looking for is a Field that will let them change the colour of the plot.

Though I suppose basic usage is just the `DefaultEditor`. Perhaps this is fine.

Reply:
Yes, it is pretty heavy but I've found it helpful to have such a thing in the docs somewhere: a straightforward statement of the architecture that people can refer to and say "yep, that about covers it". It's not for everyone but some people appreciate it. 

I do think that basic usage is the `DefaultEditor` and that anything beyond that will need to get to this level of detail. Certainly I was not able to build a simple editor without it. If you try to build an editor with just a Field to change a colour, you will need to have *at least* a layout-connected container, which you must (currently) manually construct out of `connectLayoutToPlot()` and `Panel` (because we don't export e.g. `LayoutPanel`). We can probably make that easier by exporting more stuff out of the box which might lighten up the docs, but I figured I'd start by documenting what we had and then iterating to simplify :)

Reply:
(we can discuss in #237)